docs/glossary.rst

```diff
@@ -1,2 +1,58 @@
 Glossary
 ========
+
+.. glossary::
+
+    :class:`~pulpcore.plugin.models.Artifact`
+        A file that belongs to a :term:`content unit<content>`.
+
+    :class:`~pulpcore.plugin.models.Content`
+        The smallest that of data that is managed by Pulp. When singular, "content unit" should be
+        used. Content is added and removed to :term:`Repositories<repository>`, and can have multiple
+        :term:`artifacts<artifact>`. Each content unit has a :term:`type` (like .rpm or .deb) which
+        that is defined by a :term:`plugin`
```

Comment:
"The smallest ___ of data that is managed by Pulp."  Not "that".  Perhaps "piece", "amount", or "unit"?

"Content is added to and removed from Repositories" is better.

"[...] and can have multiple artifacts." <-- should we add "or none" or maybe "can have zero, one, or more artifacts" for clarity?  On the concepts.html page it says "Each content unit can have 0 or many artifacts"

In the last sentence, choose "which" or "that" but not both.